vespalib/src/vespa/vespalib/util/threadstackexecutorbase.h

```diff
@@ -56,13 +57,14 @@ class ThreadStackExecutorBase : public SyncableThreadExecutor,
     };
 
     struct Worker {
-        Monitor    monitor;
+        std::mutex              monitor;
```

Comment:
prefer 'lock' over 'monitor' (lock/cond/guard seems to match what is done elsewhere in our code)

Reply:
Fixed